### PR TITLE
Change info message for readme in repo

### PR DESCRIPTION
**Pull request by bernt-matthias**
not sure if we should check for correct markdown.

### CODE REVIEW

File: planemo/shed_lint.py [hunk 1]
```diff
@@ -159,11 +159,6 @@ def lint_readme(realized_repository, lint_ctx):
         if os.path.exists(readme):
             readme_found = readme
 
-    readme_md = os.path.join(path, "README.md")
```
Comment (mvdbeek) [inline planemo/shed_lint.py:162]: You'll have to adjust the `test_invalid_repos` test as well.

**Comment by nsoranzo** [comment]
@mvdbeek To answer your question from Matrix, a README.rst is rendered on the TS 1.0 (see e.g. https://toolshed.g2.bx.psu.edu/repository?repository_id=552adf1ac7cd330f&changeset_revision=18c0086450bb ), while a README.md is not (see e.g. https://toolshed.g2.bx.psu.edu/repository?repository_id=9be0eab5a3639fee&changeset_revision=74810db257cc ).
I cannot check on the TTS (i.e. TS 2.0) since any repository details page loads forever for me today.

**Comment by mvdbeek** [comment]
What I mean is, this isn't displayed at all in the install process in Galaxy, so you wouldn't ordinarily find it.

**Comment by mvdbeek** [comment]
(and having a markdown readme is anyway a bit irrelevant as a check, you should be free to have one whether or not you also gave a RST file)

**Review by mvdbeek — APPROVED** [review]

**Comment by nsoranzo** [comment]
I think a tool developer may find the `"No README found skipping."` surprising if they have included a README.md and don't know about TS not supporting it. I'll propose a different wording at info level.

**Comment by nsoranzo** [comment]
What about:
```diff
diff --git a/planemo/shed_lint.py b/planemo/shed_lint.py
index 38f4ffd4..322d1675 100644
--- a/planemo/shed_lint.py
+++ b/planemo/shed_lint.py
@@ -159,16 +159,15 @@ def lint_readme(realized_repository, lint_ctx):
         if os.path.exists(readme):
             readme_found = readme
 
-    readme_md = os.path.join(path, "README.md")
-    if not readme_found and os.path.exists(readme_md):
-        lint_ctx.warn("Tool Shed doesn't render markdown, " "README.md is invalid readme.")
-        return
-
     if not readme_found:
         # TODO: filter on TYPE and make this a warning if
         # unrestricted repository - need to update iuc standards
         # first though.
-        lint_ctx.info("No README found skipping.")
+        readme_md = os.path.join(path, "README.md")
+        if os.path.exists(readme_md):
+            lint_ctx.info("Found README in Markdown format, which is not rendered by the Tool Shed, skipping")
+        else:
+            lint_ctx.info("No README found, skipping.")
         return
 
     if readme_found.endswith(".rst"):
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**